README.md

```diff
@@ -28,7 +28,12 @@ After going to our checklist above and making sure you backed everything up, we'
 If you did all of the above you may now follow these install instructions to setup a new Mac.
 
 1. Update macOS to the latest version with the App Store
-2. [Generate a new public and private SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and add them to Github
+2. [Generate a new public and private SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and add them to Github by running `ssh.sh` and following the instructions
+
+   ```zsh
+   curl https://raw.githubusercontent.com/driesvints/dotfiles/HEAD/ssh.sh | bash
```

Comment:
@driesvints I changed the script to accept an `$1` argument, but I'm not experienced enough with bash to know the recommended way of passing arguments here

Care to share your opinion?
Google is giving me a lot of different options 😅

Reply:
Doesn't this work?

```
curl https://raw.githubusercontent.com/driesvints/dotfiles/HEAD/ssh.sh | bash john@example.com
```

Reply:
Found it!

Reply:
Nice!